### PR TITLE
Scaffolding is enabled for Project + Folder

### PR DESCRIPTION
**Pull request by nosami**
Changed from Folder + File

Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1027657

### CODE REVIEW

**Comment by nosami** [comment]
@monojenkins backport release-8.4

File: main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Commands/ScaffoldCommandHandler.cs [hunk 1]
```diff
@@ -52,12 +52,12 @@ public void Scaffold ()
...
+			info.Enabled = info.Visible = NodeIsProjectOrFolder(CurrentNode) && IsAspNetCoreProject (project);
 		}
 
-		static bool NodeIsFileOrFolder (ITreeNavigator node)
-		{
-			return node.DataItem is ProjectFile || node.DataItem is ProjectFolder;
+		static bool NodeIsProjectOrFolder (ITreeNavigator node)
+		{
+			return IdeApp.ProjectOperations.CurrentSelectedItem is Project || node.DataItem is ProjectFolder;
 		}
```
Comment (mrward) [inline main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Commands/ScaffoldCommandHandler.cs]: Can you use `node.DataItem is Project`? Or does that not work?

Comment (nosami) [reply main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Commands/ScaffoldCommandHandler.cs]: Let me see - I tried to copy what RunEntryHandler did originally but that was more code.

Comment (nosami) [reply main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Commands/ScaffoldCommandHandler.cs]: Yeah. That works as well. I'll change it

**Comment by nosami** [comment]
@monojenkins backport release-8.4

**Pull request merged**